packages/common/src/styles/theme.ts

```diff
@@ -52,6 +58,11 @@ const themeOptions = {
       menu: '#f2f2f5',
       white: '#fff',
     },
+
+    form: {
+      field: '#555770',
```

Comment:
not entirely convinced that this was a good idea

Reply:
Not sure how much these (field/labels) will be used. You'd think a lot, so probably is!